src/components/Settings/System/CSystemEditorAuthBgScreen.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <b-card
+    data-test-id="card-edit-authentication"
+    class="shadow-sm"
```

Comment:
Class should be last of the options (except if style exists that should be last)
And after that there should only be events (@)